apps/site/app/[locale]/layout.tsx

```diff
@@ -10,7 +10,7 @@ import { availableLocalesMap, defaultLocale } from '@/next.locales.mjs';
 import { LocaleProvider } from '@/providers/localeProvider';
 import { ThemeProvider } from '@/providers/themeProvider';
 
-import '@/styles/index.css';
+import '@node-core/ui-components/styles/index.css';
```

Comment:
I think we still should have website-specific styles. So that the ui-components has all glkobal styles for tailwind and component-specific styles such as effects and etc...

But then the website has its own styles like body, other globals, and website-specific stuff.